src/Backend/OnboardingInsuranceAPI/Areas/User/UploadImage.cs

```diff
@@ -1,15 +1,13 @@
 using Azure.Storage.Blobs;
 using HttpMultipartParser;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Mvc;
 using OnboardingInsuranceAPI.Services;
 using System;
 using System.IO;
 using System.Threading.Tasks;
 
 namespace OnboardingInsuranceAPI.Areas.User;
 
-public class UploadUserImageHandler : IHandler
+public class UploadImage : IHandler
 {
     public async Task<string> SaveImageToBlobContainer(FilePart file)
     {
```

Comment:
Why not use 'var' ?

Reply:
var connection etc

Reply:
'var restr'? also name of this variable is really bad.

Reply:
Ok